code/modules/clothing/gloves/color.dm

```diff
@@ -130,21 +130,35 @@
 
 /obj/item/clothing/gloves/color/latex
 	name = "latex gloves"
-	desc = "Cheap sterile gloves made from latex."
+	desc = "Cheap sterile gloves made from latex. Transfers minor paramedic knowledge to the user via budget nanochips."
 	icon_state = "latex"
 	item_state = "lgloves"
 	siemens_coefficient = 0.3
 	permeability_coefficient = 0.01
 	transfer_prints = TRUE
 	resistance_flags = NONE
 
+/obj/item/clothing/gloves/color/latex/equipped(mob/user, slot)
+	if(slot == SLOT_GLOVES)
+		ADD_TRAIT(user, TRAIT_QUICK_CARRY, CLOTHING_TRAIT)

```

Comment:
does this need inher? (dropped too)

Reply:
same for others

Reply:
beep

Reply:
whoops

Reply:
i don't know what you mean by inher, but i tested it and it works just fine, it adds and removes the traits depending on the gloves you wear and take off

Reply:
nevermind, you can't firecarry them without gloves,it gives a 'they're standing!' message, and it apparently does not in fact remove the trait when dropped

pleas help

Reply:
fairly sure i fixed the first error, but not sure about the second

Reply:
@ExcessiveUseOfCobblestone fixed both issues, 99% sure the pr is good now

Reply:
inher is ..()